jvcl/run/JvXPCoreUtils.pas

```diff
@@ -53,15 +53,15 @@ procedure JvXPDrawBoundLines(const ACanvas: TCanvas; const BoundLines: TJvXPBoun
 
 procedure JvXPConvertToGray2(Bitmap: TBitmap);
 procedure JvXPRenderText(const AParent: TControl; const ACanvas: TCanvas;
-  ACaption: TCaption; const AFont: TFont; const AEnabled, AShowAccelChar: Boolean;
+  ACaption: TCaption; const AFont: TFont; const AEnabled: Boolean; const AAccelCharType: TJvXPAccelChar;
```

Comment:
This will break all existing users code, on top of maybe some JVCL one as well.
What's the point of such a breaking decision?
Wouldn't it be possible to create an overload with the original signature that calls the new signature?